playwright.config.ts

```diff
@@ -87,6 +87,16 @@ const config: PlaywrightTestConfig = {
       : 'on-first-retry',
     screenshot: 'only-on-failure',
     video: 'off',
+    storageState: {
+      // @ts-expect-error The options have been marked as mandatory however the except the same arguments as https://playwright.dev/docs/api/class-browsercontext#browser-context-add-cookies
```

Comment:
There is an issue with the types.  All the options have been marked as mandatory, it seems to use this type internally https://github.com/microsoft/playwright/blob/27027658dc208a5202944313a310f9571a51e5ac/packages/playwright-core/src/client/network.ts#LL49C6-L49C6